src/main/java/io/localmotion/chatbox/ChatMessage.java

```diff
@@ -28,7 +30,9 @@
 
     private final Date creationTime = new Date();
 
-    @NotBlank
+    @NotBlank(message = "Message must have at least " + SmokefreeConstants.ChatBox.MINIMUM_MESSAGE_LENGTH + " characters")
+    @Pattern(regexp = "^[A-Za-z0-9\\n!@&(),.?\": ]+$", message = "Please enter only allowed special charaxters: @&(),.?\": ")
```

Comment:
Drop the regexp. We will trust (and test) the escaping of React.

Reply:
I have removed regexp